source/draw/FloatImage.ooc

```diff
@@ -3,13 +3,13 @@ use ooc-math
 FloatImage : class {
 	// x = column
 	// y = row
-	_size: IntSize2D
+	_size: IntVector2D
 	size ::= _size
 	_pointer: Float*
 	pointer ::= this _pointer
-	init: func ~IntSize2D (=_size)
+	init: func ~IntVector2D (=_size)
```

Comment:
You are right that one of them _could_ drop the suffix. It seams to be the norm in the Image classes to have suffixes on all initiators.